challenges/classes.js

```diff
@@ -1,7 +1,26 @@
 // 1. Copy and paste your prototype in here and refactor into class syntax.
 
+class Cuboid {
+    constructor(attr) {
+        this.length = attr.length;
+        this.width = attr.width;
+        this.height = attr.height; 
+    }
+    volume() {
+        return (this.length * this.width * this.height);
+    }
+    surfaceArea() {
+        return (2 * (this.length * this.width + this.length * this.height + this.width * this.height));
```

Comment:
return (2 * (`${this.length}` * `${this.width}` + `${this.length} * `${this.height}` + `${this.width}` * `${this.height}`));